### ISSUE TITLE
[1.12.1] Freeze while placing projector

### ISSUE THREAD

**Issue by UnPure** [CompactMods/CompactMachines#229]
Hey there.
I'm trying to build a 9x9x9 Cube. Whenever i place the last projector around the cube, the game is freezing.
After restarting the game and rejoining, its freezing again, only a backup can save the world.
Because the game is just freezing and not crashing, i dont have a crashlog.

**CompactMachines:** 3.0.0-b127
**Forge:** 14.22.0.2475

The projectors are placed with a 1 block space to the cube and 2 blocks to the ground.

A full list of all used mods can be found [here](http://upclan.de/modpack/info/)

If you need more informations, let me know!

**Comment by davenonymous** [comment]
Well, I need to be able to reproduce this to fix it. I can not reproduce it so far.
Any idea what I need to do to trigger this behaviour?
Are you placing the projectors after or before you've placed the actual cube?

"9x9x9" and "1 block space to the cube" sounds very wrong though - the projector can't project a 9x9 wall one block in front of it.

**Comment by UnPure** [comment]
Ok so we tested more and more and it seems like we found the issue.
It seems to be a compat issue with HWYLA (1.8.20-B35).
Sometimes if you place the projector, its facing in the wrong direction instead of the direction you are looking at. If that happens, its freezing the client.
This also happens with every other recipe, we are using the tunnel recipe to test that.

For the 9x9x9, i know thats wrong, but its still freezing for the same reason.^^
We are placing the projector after creating the structure.
